wwwroot/js/site.js

```diff
@@ -1,4 +1,9 @@
-﻿// Please see documentation at https://learn.microsoft.com/aspnet/core/client-side/bundling-and-minification
-// for details on configuring this project to bundle and minify static web assets.
-
-// Write your JavaScript code.
+﻿// Please see documentation at https://learn.microsoft.com/aspnet/core/client-side/bundling-and-minification
+// for details on configuring this project to bundle and minify static web assets.
+
+// Write your JavaScript code.
+
+function rentBookConfirm(bookId) {
+    window.location.href = 'https://localhost:7095/Books/Details?id=' + bookId;

```

Comment:
Bỏ địa chỉ để app có thể hoạt động trên mọi cổng port của localhost.
```suggestion
    window.location.href = '/Books/Details?id=' + bookId;
```